extension/content.js

```diff
@@ -294,6 +294,32 @@ function markMergeCommitsInList() {
 	});
 }
 
+function indentInput(el, size = 4) {
+	el.focus();
+	const val = el.value;
+	if (typeof el.selectionStart === 'number') {
```

Comment:
no is not.. you are right... I refactor this code for a snippet from stackoverflow, ant I think it had support for IE11, that was a check for that...